src/components/Flows/__tests__/SingleFlowRunHistory.test.tsx

```
@@ -12,41 +13,82 @@
     expect(logmessages).toBeInTheDocument();
+    expect(screen.getByText('flow-run-lastRun')).toBeInTheDocument();
+    expect(screen.getByTestId('single-flow-run-logs')).toBeInTheDocument();
+  });
+
+  it('renders safely when no flowRun is provided', () => {
+    renderComponent(null);
+    expect(screen.getByTestId('single-flow-run-logs')).toBeInTheDocument();
+  });
+
+  it('calls errorToast when fetchLogs fails', async () => {
+    const { httpGet } = require('@/helpers/http');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)